src/ReportPortal.Client/Abstractions/Resources/ILogItemResource.cs

```diff
@@ -17,51 +17,85 @@ public interface ILogItemResource
         /// <param name="request">Information about representation of log item.</param>
         /// <returns>Representation of just created log item.</returns>
         //Task<LogItemCreatedResponse> CreateAsync(CreateLogItemRequest request);
-
         Task<LogItemCreatedResponse> CreateAsync(CreateLogItemRequest request);
 
+        /// <inheritdoc cref="CreateAsync(CreateLogItemRequest)"/>
+        /// <param name="cancellationToken">Cancellation token.</param>
+        Task<LogItemCreatedResponse> CreateAsync(CreateLogItemRequest request, CancellationToken cancellationToken);
+
         /// <summary>
         /// Creates a new log item.
         /// </summary>
         /// <param name="requests">Information about representation of log item.</param>
         /// <returns>Representation of just created log item.</returns>
         //Task<LogItemCreatedResponse> CreateAsync(CreateLogItemRequest request);
-
         Task<LogItemsCreatedResponse> CreateAsync(params CreateLogItemRequest[] requests);
 
+        /// <inheritdoc cref="CreateAsync(CreateLogItemRequest[])"/>
+        /// <param name="cancellationToken">Cancellation token.</param>
+        Task<LogItemsCreatedResponse> CreateAsync(CancellationToken cancellationToken, params CreateLogItemRequest[] requests);
```

Comment:
@nvborisenko maybe it makes sense to use simple array of CreateLogItemRequests instead of params? But it can be breaking changes.

Reply:
Good point, makes sense as for me.

Reply:
We have already had it:
![image](https://user-images.githubusercontent.com/103627305/173188272-da779ad5-6155-43dd-9473-8694bbdb6edd.png)

Reply:
Good to see it in place already, thanks.